public/index.html

```diff
@@ -150,14 +150,32 @@
             <p>Conferences are not only about listening. Most of the value comes from talking to each other. Our mission
                 is to create an inspiring, thoughtful, creative and safe space to everyone involved. <b>The venue</b> is
                 known to be of good quality - it’s a University of Wroclaw building. It has many ways to ensure safety
-                of all the people inside like monitoring and special security people.<br/>As part of our goal, we’re
-                dedicated to react to all situations that we’ll be notified, ideally directly to us.There will be a <b>special
-                    phone number</b>, you can call at any time to let us know about any problem or ask any
-                question.<br/>Additionally, there will be a special team of <b>student volunteers</b> who will be
-                visible during the conference (If you want to join the team, please let us know). Student volunteers are
-                here to help find your way around and resolve any problem. It is needless to say that this conference is
+                of all the people inside like monitoring and special security people.<br/> It is needless to say that this conference is
                 a place for good people, only. Good people respect each other, are nice to each other, smile, and make
                 everyone (without exceptions) feel comfortable.</p>
+            <body>
+            <h3>Conference Code of Conduct</h3>
+
+            <p>All attendees, speakers, sponsors and volunteers at our conference are required to agree with the following code of conduct. Organisers will enforce this code throughout the event. We are expecting cooperation from all participants to help ensuring a safe environment for everybody.</p>
```

Comment:
Good point, but ii`m not sure if it is okey to add this point, when ppl bought tickets.